src/Simulation/NativeSparseSimulator/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 	target_compile_options(Microsoft.Quantum.SparseSimulator.Runtime PUBLIC -O3 -ftree-vectorize -mavx2 -mfma)
 endif()
 
+if (MSVC)
+    add_compile_options(/bigobj)
+endif()
```

Comment:
How does this relate to the issue being fixed?
Code comments wouldn't harm.

Reply:
Normally we build this with clang. Building with cmake/MSVC will fail without this switch.

Reply:
Why would one need to build it with MSVC? To debug easier? And just for local builds (not CI)?
If yes then I would recommend to comment this out and provide extra comments explaining why one may need this.

Reply:
Moving to a separate PR.